pyttman/core/internals.py

```diff
@@ -60,10 +69,20 @@ def __init__(self, **kwargs):
         self.LOG_FORMAT: str | None = None
         self.LOG_TO_STDOUT: bool = False
 
-        [setattr(self, k, v) for k, v in kwargs.items()
+        [self.__set_attr(k, v) for k, v in kwargs.items()
          if not inspect.ismodule(v)
          and not inspect.isfunction(v)]
 
+    def __set_attr(self, k, v):
```

Comment:
According to PEP-8, single `_` hint to a private method - unless mangling is required, but it' highly unlikely to be needed here.